download.mdwn

```diff
@@ -54,6 +54,7 @@ A list of of old releases [[is available here|releases]].
 * [openSUSE](http://download.opensuse.org/repositories/X11:/windowmanagers/)
 * [PLD Linux](http://pld-linux.org)
 * [Mageia](http://mageia.madb.org/package/show/application/0/name/awesome)
+* [OpenIndiana](http://pkg.openindiana.org/hipster/en/search.shtml?token=awesome&action=Search)
```

Comment:
I'm ok with linking to the search. I looked briefly around, but did not really find any better link target. If it breaks, we just fix the link (once we noticed that it broke...).